infra/base-images/base-runner/run_fuzzer

```diff
@@ -45,6 +45,13 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
   # AFL expects at least 1 file in the input dir.
   echo input > /tmp/input/input
   CMD_LINE="$OUT/afl-fuzz $AFL_FUZZER_ARGS -i /tmp/input -o /tmp/afl_output $@ $OUT/$FUZZER"
+elif [[ "$FUZZING_ENGINE" = honggfuzz ]]; then
+  # Honggfuzz expects at least 1 file in the input dir.
+  echo input > $CORPUS/input
+  rm -rf /tmp/honggfuzz_workdir && mkdir /tmp/honggfuzz_workdir
+  # --linux_keep_aslr: some bug/feature in the kernel that prevents
+  # us from doing personality(ADDR_NO_RANDOMIZE).
+  CMD_LINE="$OUT/honggfuzz --linux_keep_aslr -W /tmp/honggfuzz_workdir -v -z -P -f \"$CORPUS\" $@ -- \"$OUT/$FUZZER\""
```

Comment:
would you mind adding a short comment here explaining what all these "-v" "-z", etc arguments are doing?